config.yaml

```diff
@@ -59,23 +60,20 @@ contact: 'team@carpentries.org'
 
 # Order of episodes in your lesson
 episodes: 
-- 01-getting-started.md
-- 02-variables.md
-- 03-types-conversion.md
-- 04-built-in.md
-- 05-coffee.md
-- 06-libraries.md
-- 10-lunch.md
-- 11-lists.md
-- 12-for-loops.md
-- 13-looping-data-sets.md
-- 14-writing-functions.md
-- 15-scope.md
-- 16-coffee.md
-- 17-conditionals.md
-- 18-style.md
-- 19-wrap.md
-- 20-feedback.md
+- getting-started.md
+- variables.md
+- lists.md
+- functions.md
+- libraries.md
+- for-loops.md
+- looping-data-sets.md
+- pandas.md
+- conditionals.md
+- writing-functions.md
+- tidy.md
+- data-visualisation.md
+- wrap.md
```

Comment:
This is most likely the reason why you could not automatically sync your fork with the LC lesson on GitHub: a large number of the files have changed name, which has a tendency to confuse Git.